webapp/store/survey/actions/active.js

```diff
@@ -0,0 +1,12 @@
+import axios from 'axios'
+
+import { surveyCreate, surveyUpdate } from './actionTypes'
+
+export const setActiveSurvey = (surveyId, canEdit = true, dispatchSurveyCreate = false) => async (dispatch) => {
```

Comment:
How about use the params as objects to avoid problems with the order?
({ surveyId, canEdit, ...})